frb/data/Galaxies/180924/FRB180924_host.json

```diff
@@ -61,58 +61,58 @@
         "[OII] 3729_err": 2.5078099684377433e-18
     },
     "offsets": {
-        "ang_avg": 0.7066782158974648,
-        "ang_avg_err": 0.11551890797698443,
+        "ang_avg": 0.7229508018365346,
+        "ang_avg_err": 0.1740307054526543,
         "ang_best": 0.7000918119916533,
-        "ang_best_err": 0.11570652019918955,
+        "ang_best_err": 0.17552555328805247,
         "physical": 3.369230808914334,
-        "physical_err": 0.5568440681206338
+        "physical_err": 0.8447264940971653
     },
     "photom": {
         "DES_Y": 19.789268983065682,
         "DES_Y_err": 0.056431600000000005,
-        "DES_Y_flux": 4.408515794288702e-05,
-        "DES_Y_flux_err": 2.351936804302065e-06,
+        "DES_Y_flux": 0.044085157942887025,
```

Comment:
Do we understand why these values are changing by orders of magnitude?
We had better..  :)

Reply:
Yeah, I think Sunil did some modifications in the wrapper. Can you confirm @SunilSimha?

Reply:
According to my git logs, I had fixed a flux unit conversion error (see commit f5c64c5). This is just the result of that.

Reply:
roger that.